cmd/create.go

```diff
@@ -153,8 +151,7 @@ func createGenesis(cmd *cobra.Command, args []string) error {
 			}
 			subnetType = models.VmTypeFromString(subnetTypeStr)
 		}
-		err = createSidecar(subnetName, subnetType)
-		if err != nil {
+		if err = createSidecar(subnetName, subnetType, ""); err != nil {
```

Comment:
why not ask for token name for the case the genesis is being given?

Reply:
good idea, I'll address when I make my follow-up PR focusing on the tokenName